tests/e2e/runner/runner.go

```diff
@@ -97,20 +88,34 @@ done:
 
 		outf("{{magenta}}checking custom VM status{{/}}\n")
 		cctx, ccancel := context.WithTimeout(context.Background(), 2*time.Minute)
```

Comment:
Is there any reason that we create a separate context here for this request rather than the first one created in `WaitForCustomVm(...)` ?

Reply:
I think it's because the first timer has a significantly longer timeout than the healthcheck